Writerside/topics/Pull-request-insights.md

```diff
@@ -48,6 +48,7 @@ The visual image is attached to the same pull request comment which contains the
 
 <img src="code-quality-metrics-on-pr.png" alt="pull request insights code quality image " border-effect="line" width="700" 
 thumbnail="false"/>
+<note>Requires that your organization installs our GitHub Action.</note>
```

Comment:
Change verbiage to `Astronuts GitHub Code Quality Action` and reference it as a hyperlink to the relevant page. See existing doc pages for reference.  This is to maintain a consistent pattern.

Reply:
Done